static/docs/use-cases/huge-data-on-an-external-drive.md

```diff
@@ -0,0 +1,156 @@
+# Huge Data On An External Local Drive
+
+Sometimes the data may be huge and they are stored on an external local drive.
+By "huge" we mean that they won't fit on our home directory, and even if they
```

Comment:
Maybe point out that not even the entire root drive (not sure that's the best term) is too small for the data files in question.

Reply:
Any comments here?

Reply:
My point here was that we probably shouldn't assume the workspace is in the home directory. We don't really use the home directory in other docs. Maybe just talk about the "root drive" or "local file system"?

Reply:
Changing.